docs/state.md

```diff
@@ -236,7 +236,47 @@ AppStore
     'tabs.switch-to-new-tabs': boolean, // true if newly opened tabs should be focused immediately
     'tabs.tabs-per-page': number // number of tabs per tab page
   },
-  sites: {
+  // TODO sort alphabetically
```

Comment:
this todo only means that I should reorder this entities in this `md` file when all properties are fixed